DFE-Digital/check-your-eligibility-frontend
# Jcb review page widths 190566

QA Note: Parent Journey

When implementing this fix it became clear that we needed a more comprehensive way to manage the layout of views across all journeys. There are two main widths currently in use across the system: two-thirds, and full.

In general we would use two-thirds for forms and general content. For tabular data such as search results we would use full to gain the extra width on the screen to fit more content. The bug in this ticket is with the loader page which should be full width.

The code has been refactored in the layout so that the view can pass a parameter detailing which view to use and then the appropriate width is used for that page when it is rendered. Doing this meant adding the new code to the Layout and the parameter to each view. The method is that if the parameter is "two_thirds" then it uses that width else it uses full. So, for example, if a page didn't send a value the default would be full width.

I have designated the majority of pages in the application as two thirds based off the content in the prototype so you can presume all pages have been updated in this way. Listed below are all the pages that I set as full width so that QA have a reference of the ones that should look different.

- Loader.cshtml
- Application_Sent.cshtml

Due to the Loader page becoming full width and the outcome pages being injected into the loader page container to display them, and them all being two-thirds width, all outcomes have also been edited and should be tested to ensure they show as two-thirds as the layout width for them is specified within the view itself.

- Could_Not_Check.cshtml
- Eligible.cshtml
- Not_Eligible.cshtml
- Not_Found.cshtml
- Not_Found_Pending.cshtml
- Technical_Error.cshtml

QA Note: Parent Journey

Pages designated as full-width but for School/LA Journey:

- Process_Appeals.cshtml
- Loader.cshtml (added parameter but display's as before)
- Check_Answers.cshtml
- ApplicationsRegistered.cshtml
- AppealsApplications.cshtml
- ApplicationDetail.cshtml
- ApplicationDetailAppeal.cshtml
- ApplicationDetailFinalise.cshtml
- ApplicationDetailLa.cshtml
- FinaliseApplications.cshtml
- PendingApplications.cshtml
- Search.cshtml
- SearchResults.cshtml

​As with the Parent journey the outcomes for School and LA have been updated. (Remember that Eligible and Not_Eligible are separate pages for LA journey).

- Eligible.cshtml
- Eligible_LA.cshtml
- Not_Eligible.cshtml
- Not_Eligible_LA.cshtml
- Not_Found.cshtml
- `Technical_Error.cshtml
